packages/in_app_purchase/in_app_purchase_ios/CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
+## 0.2.0+1
+
+* Fix CHANGELOG for 0.2.0 (added missing information regarding the implementation of promotional offers).
```

Comment:
Fixes

(Per the [new style guide](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style))

Reply:
Fixed.